src/coreclr/src/vm/methodtablebuilder.h

```diff
@@ -2282,7 +2284,8 @@ class MethodTableBuilder
     class DeclaredMethodIterator
     {
       private:
-        MethodTableBuilder &m_mtb;
+        int                 m_numDeclaredMethods;
```

Comment:
Can `m_numDeclaredMethods` be marked const?

Reply:
Yes. It doesn't have any effect on codegen, but these two fields could be marked const.